### DISCUSSION TITLE
Can someone explain [blogrelated] and [wikirelated], do they work as expected?

### DISCUSSION THREAD

**Discussion by emmanuego** [datenstrom/community#498] [Ask a question] [answered]
Hello,
First, consider english is not my first language, so maybe there is something I misunderstand.

In the help section of wiki there is this line : "[wikirelated] for a list of pages, related to the current page"
But when I use [wikirelated /wiki/ ] on a page it shows a list of all the wiki pages of my wiki, except the current one.
I was expected that it shows a list of pages that have links to the current pages

Same problem for [blogrelated]

PS : i'm running Yellow in Wamp

**Comment by schulle4u** [comment, marked as answer]
The `blogrelated` and `wikirelated`  shortcuts use title, author and tag information from other wiki and blog pages to determine if they are related to the current page. You can use the `pagesMax` parameter to limit the output, default is 5 pages. 

Listing all pages that link to the current page like e.g. in Mediawiki currently is not supported, but sounds like an idea to consider working on.

HTH
Steffen

**Comment by emmanuego** [comment]
Ok, thank you for your explanation
